core/kernel/notif.c

```diff
@@ -263,6 +263,25 @@ void notif_it_set_mask(unsigned int id, bool do_mask)
 		set_it_mask(id, do_mask);
 }
 
+/*
```

Comment:
We're normally documenting the interface in the .h files.

Reply:
ok